plasmapy/diagnostics/thomson.py

```diff
@@ -557,6 +583,7 @@ def spectral_density(
         electron_vel=electron_vel.to(u.m / u.s).value,
         probe_vec=probe_vec,
         scatter_vec=scatter_vec,
+        scattering_angle=scattering_angle,
```

Comment:
Following my other comments, you would then delete the probe_vec and scatter_vec keywords here and always just pass in the scattering angle (regardless of how it was calculated above)